### PR TITLE
Feat/invitations

### PR DESCRIPTION
**Pull request by ignaciomansuy**
## Description

1. Added invitation form component. It's a modal, does the post to bakend (for now mockServer), mini spinner when waiting for fetch response and puts message "Invitation sended!" for 5 seconds when invitation is created. Reusable for different entities.
2. Added button "Add colaborator" to organization/:id view and project/:id. Opens Invitation form modal.
3. Redesigned project/:id view to list subprojects in it. Done UI for mobile and desktop format. 

TODO (by me): add button in subproject card (in projects/:id view) to open the 3D model viewer on a modal component.

## Requirements

None

## Additional changes

1. Fixed hooks and services to call API and mockserver. Previous config resulted in error with base Url when trying to use both functions in same page
2. Deleted dead pages and components
3. Made custome RadioButton component
4. Added type prop to PrimaryButton component
5. Added props to Spinner so it can be used as a tiny spinner

### CODE REVIEW

File: pages/projects/[id].js [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+
   const router = useRouter()
   const [projectData, isLoading, error] = useFetch({
     url: `/projects/${router.query.id}`,
     method: "get",
   })
+  const current_path = `/projects/${router.query.id}/subprojects`
+
+  // /projects/:id should return subprojects with eager loading
+  const [subprojects, isLoading2] = useMockFetch({
```
Comment (NicoMeyer) [inline pages/projects/[id].js:31]: Here I would use a more descriptive name, so we can know it's related to the subprojects.

File: components/shared/InvitationForm.jsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      accessType: "r",
+    },
+    mode: "onChange",
+  })
+
+  const [errorMessage, setErrorMessage] = useState(null)
+  const [uploadMessage, setuploadMessage] = useState(false)
+  const [loading, setLoading] = useState(null)
+
+  const radioButtons = {
```
Comment (NicoMeyer) [inline components/shared/InvitationForm.jsx:27]: I would save this const in the constants folder.

**Review by NicoMeyer — CHANGES_REQUESTED** [review]
All seems fine, it's good progress regarding invitation and forms. I left some minor comments. When merging rememeber to accept the url from Main in ProjectForm.

**Comment by ignaciomansuy** [comment]
Solved requested changes

**Review by NicoMeyer — APPROVED** [review]
10/10

**Pull request merged**